tests/test_cube_help.py

```diff
@@ -18,6 +18,62 @@
 
 class TestCubeHelp(unittest.TestCase):
 
+    def _generate_ocean_cube(self):
```

Comment:
To allow this method to be reused, could you pop it into tests/common.py please and import from there please? You might also need to add an empty __init__.py file to tests to turn it into a module and to allow the import.